_config.yml

```diff
@@ -16,10 +16,10 @@
 title: Code4Lib
 # email: your-email@example.com
 description: >- # this means to ignore newlines until "baseurl:"
-  We are developers and technologists for libraries, museums, and archives who are dedicated to 
+  We are developers and technologists for libraries, museums, and archives who are dedicated to
   being a diverse and inclusive community, seeking to share ideas and build collaboration.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "http://code4lib.org" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://code4lib.org" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
I don't know what the consequence of this is. The site loads fine over HTTP on the local dev server and I doubt we have mixed content embedded anywhere. But if something goes haywire after merging maybe take a look at the HTTPS here